app/styles/components/_pdf-button.scss

```diff
@@ -0,0 +1,21 @@
+.pdf-button-container {
+    text-align: center;
+
+    .pdf-button {
+        border: 0;
+        height: $button-large-height;
+        margin: $aside-margin;
+        padding: 5px;
+        color: white;
+        background-color: $gray-darker;
+    }
+
+    .pdf-button:hover {
```

Comment:
Since you're using SASS, you can use http://sass-lang.com/documentation/file.SASS_REFERENCE.html#parent-selector

For example:
```
.pdf-button {
    // bunch of other styles

    &:hover {
        cursor: pointer;
    }
}
```